### PR TITLE
Cleaned up dead code and run modes

### PR DESCRIPTION
**Pull request by kalysbek-z**
Deleted not related to VS topic functions and their run modes.

### CODE REVIEW

File: src/interface.cpp [hunk 1]
```diff
@@ -33,38 +27,28 @@ int main(int argc, char *argv[]){
...
+      default : return(0);
     	}
   	}
 
-  if (PARAMS.verbosity == 2) viewParams(&PARAMS);
-
+//  if (PARAMS.verbosity == 2)
 
   // -- run the system depending on runMode
   switch(PARAMS.runMode){
```
Comment (talkanbaev-artur) [inline src/interface.cpp:37]: Remove all run modes except vs - we do not need a switch here

File: src/gpuCode.h [hunk 2]
```diff
@@ -42,20 +42,10 @@ int updatePopulation(GPU_Palette* P, AParams* PARAMS, int day);
 // kernel calls:
 __global__ void setMap(float* map, float val, long sizePopulation);
 
-
-
-
-
-// -------- runmode 1 stuff for playing with pixel colors
```
Comment (talkanbaev-artur) [inline src/gpuCode.h:49]: Is it only runmode 1? Please double check with old versions and check if vs works. We need this build to be compilable

**Review by talkanbaev-artur — CHANGES_REQUESTED** [review]

File: src/gpuCode.h (hunk 2 above)
Comment (kalysbek-z) [reply src/gpuCode.h:49]: it works. Yes, there was only run mode 1

Comment (talkanbaev-artur) [reply src/gpuCode.h:49]: How does it work? When you launch it is a black screen, which slowly becomes brighter?

File: src/hostCode.cpp [hunk 3]
```diff
@@ -31,10 +31,14 @@ int runVS(AParams* PARAMS){
 
   GPU_Palette P1 = initPopulation();
 
+  printf("1\n");
+
   CPUAnimBitmap animation(PARAMS->width, PARAMS->height, &P1);
   cudaMalloc((void**) &animation.dev_bitmap, animation.image_size());
   animation.initAnimation();
 
+    printf("1\n");
```
Comment (talkanbaev-artur) [inline src/hostCode.cpp:40]: :moyai:

**Review by talkanbaev-artur — APPROVED** [review]
LGTM!

**Pull request merged**